tasks/main.yml

```diff
@@ -6,6 +6,7 @@
   package:
     name: procps
   ignore_errors: yes
+  register: ignore_errors_register
```

Comment:
What is the purpose of this register?

Reply:
My linter refused an `ignore_errors: yes` and ansible-lint proposed this addition. I know this depends on the version of the linter and could be avoided right now but it does not break anything